frontend/public/style/_overrides.scss

```diff
@@ -32,6 +32,9 @@ $pf-4-nav-bar-height: 76px; // Height of the PatternFly 4 masthead
   margin-left: 3px;
 }
 
+.pf-c-button-icon--plain {
```

Comment:
This is an odd place to put this.  Perhaps alphabetize it in the PF4 overrides below?